### PR TITLE
Add cudnn and cutensor for CUDA 11.2 in `install_library`

### PR DESCRIPTION
**Pull request by emcastillo**
Needed by https://github.com/cupy/cupy-release-tools/pull/66

### CODE REVIEW

**Comment by kmaehashi** [comment]
pfnCI, test this please.

cc/ @takagi This is part of CUDA 11.2 support.

File: cupyx/tools/install_library.py [hunk 1]
```diff
@@ -100,6 +104,9 @@ def _make_cutensor_record(
     }
 
 
+_cutensor_records.append(_make_cutensor_record(
+    '11.2', '1.2.2',
+    'libcutensor-linux-x86_64-1.2.2.5.tar.gz', ''))
```
Comment (leofang) [inline cupyx/tools/install_library.py:109]: Windows support is being added in https://github.com/cupy/cupy/pull/4678.

Comment (kmaehashi) [inline cupyx/tools/install_library.py:109]:
```suggestion
    'libcutensor-linux-x86_64-1.2.2.5.tar.gz',
    'libcutensor-windows-x86_64-1.2.2.5.zip'))
```

**Comment by leofang** [comment]
Jenkins, test this please

**Comment by leofang** [comment]
Test failures are also seemed when testing #4628, but I thought it only happens on CUDA 10.0...Not sure if they are flaky or not.
```
tests.cupy_tests.linalg_tests.test_decomposition.TestSVD_param_0_{full_matrices=True}.test_svd_rank3_loop
tests.cupy_tests.random_tests.test_generator.TestInterval.test_goodness_of_fit
```
https://jenkins.preferred.jp/job/chainer/job/cupy_pr/3055/

**Comment by leofang** [comment]
Jenkins, test this please

**Pull request merged**